src/Hamiltonians/ReducedDensityMatrix.jl

```diff
@@ -0,0 +1,150 @@
+"""
+    SingleParticleReducedDensityMatrix(i, j) <: AbstractHamiltonian
+
+Represent a {i,j} element of the single-particle reduced density matrix:
+
+```math
+\\hat{ρ}^{(1)}_{i,j} = \\langle \\psi | \\hat a^†_{i} \\hat a_{j} | \\psi \\rangle
+```
+
+where ``i`` and ``j`` are the `mode` and ``| \\psi \\rangle`` is the `state-ket` of the given Hamiltonian.
```

Comment:
Why are mode and state-ket in backticks? Backticks print with a monospace font and are used to refer to something in the code, i.e. function and argument names.